### PR TITLE
en: update doc for deploying TiDB cluster on ACK

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update doc for deploying TiDB cluster on Alibaba cloud.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/30
- Other reference link(s):

### CODE REVIEW

File: en/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -163,29 +217,63 @@ The initial login user account and password:
...
+> **Note:**
+>
+> You have to manually delete the cloud disk used by the components in the Alibaba Cloud console.
 
 ## Configure
 
 ### Configure TiDB Operator
 
-You can adjust the `variables.tf` settings to configure TiDB Operator. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example,
+You can set variables in `terraform.tfvars` to configure TiDB Operator. Most configuration items can be modified after understanding the semantics based on the comment of `variable`. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example:
```
Comment (DanielZhangQD) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
You can set variables in `terraform.tfvars` to configure TiDB Operator. Most configuration items can be modified after understanding the semantics based on the comments of the `variable`. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example:
```

File: en/deploy-on-alibaba-cloud.md [hunk 2]
```diff
@@ -203,7 +291,7 @@ In the default configuration, the Terraform script creates a new VPC. To use the
 
 ### Configure the TiDB cluster
 
-`./my-cluster.yaml` is the `values.yaml` configuration file in the TiDB cluster. You can configure the TiDB cluster by modifying this file. For supported configuration items, see [Configure the TiDB cluster in Kubernetes](configure-a-tidb-cluster.md).
+See [TiDB Operator API Documentation](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) and [Using TidbCluster](configure-cluster-using-tidbcluster.md).
```
Comment (DanielZhangQD) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
See [TiDB Operator API Documentation](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) and [Configuring TiDB Cluster](configure-cluster-using-tidbcluster.md).
```

File: en/deploy-on-alibaba-cloud.md [hunk 3]
```diff
@@ -334,34 +421,6 @@ You can customize this script. For example, you can remove the `module "bastion"
 >
 > You can copy the `deploy/aliyun` directory. But you cannot copy a directory on which the `terraform apply` operation is currently performed. In this case, it is recommended to clone the repository again and then copy it.
 
-## Destroy
```
Comment (DanielZhangQD) [inline en/deploy-on-alibaba-cloud.md]: Please keep this section here, no need to move to the front.

Comment (ran-huang) [reply en/deploy-on-alibaba-cloud.md]: This is to keep it consistent with the [Chinese doc](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/deploy-on-alibaba-cloud.md). Please confirm whether we should move it.

**Comment by ran-huang** [comment]
@DanielZhangQD PTAL again, thanks!

File: en/deploy-on-alibaba-cloud.md [hunk 4]
```diff
@@ -77,14 +79,26 @@ All the instances except ACK mandatory workers are deployed across availability
     cd tidb-operator/deploy/aliyun
     ```
 
-    Note that you must answer "yes" to `terraform apply` to continue:
+    You can create or modify `terraform.tfvars` to set the variables, and configure the cluster to fit your need. You can view the configurable variables and their descriptions by `variables.tf`. The following is an example of how to configure the ACK cluster name, the TiDB cluster name and the number of nodes in PD, TiKV, and TiDB.
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
    You can create or modify `terraform.tfvars` to set the values of the variables, and configure the cluster to fit your needs. You can view the configurable variables and their descriptions in `variables.tf`. The following is an example of how to configure the ACK cluster name, the TiDB cluster name and the number of PD, TiKV, and TiDB nodes.
```

File: en/deploy-on-alibaba-cloud.md [hunk 5]
```diff
@@ -77,14 +79,26 @@ All the instances except ACK mandatory workers are deployed across availability
...
+
+    After the configuration, execute the following commands to initialize and deploy the cluster:
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     terraform init
     ```
 
+    Input "yes" to confirm execution when you run the `apply` command:
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
    Input "yes" to confirm execution when you run the following `apply` command:
```

File: en/deploy-on-alibaba-cloud.md [hunk 6]
```diff
@@ -136,6 +150,44 @@ All the instances except ACK mandatory workers are deployed across availability
...
+    ```shell
+    cd manifests/ && mv db-monitor.yaml.example db-monitor.yaml && mv db.yaml.example db.yaml
+    ```
+
+    To complete the CR file configuration, refer to [TiDB Operator API documentation](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) and [Configuring TiDB Cluster](configure-cluster-using-tidbcluster.md).
+
+    > **Note:**
+    >
+    > * Replace all the `TIDB_CLUSTER_NAME` in the `db.yaml` and `db-monitor.yaml` files with `tidb_cluster_name` configured in the deployment of ACK.
+    > * Make sure the numbers of nodes in PD, TiKV, and TiDB are the same as the `replicas` field in `db.yaml`.
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
    > * Make sure the number of PD, TiKV, and TiDB nodes is the same as the `replicas` value of the corresponding component in `db.yaml`.
```

File: en/deploy-on-alibaba-cloud.md [hunk 7]
```diff
@@ -163,29 +217,63 @@ The initial login user account and password:
 
 > **Warning:**
 >
-> It is strongly recommended to set `deploy/modules/aliyun/tidb-cluster/values/default.yaml` - `monitor.grafana.service.annotations` - `service.beta.kubernetes.io/alicloud-loadbalancer-address-type` to `intranet` for security if you already have a VPN connecting to your VPC or plan to set up one.
+> If you already have a VPN connecting to your VPC or plan to set up one, it is strongly recommended to set `db-monitor.yaml` - `spec.grafana.service.annotations` - `service.beta.kubernetes.io/alicloud-loadbalancer-address-type` to `intranet` for security.
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
> If you already have a VPN connecting to your VPC or plan to set up one, it is strongly recommended that you go to the `spec.grafana.service.annotations` section in the `db-monitor.yaml` file and set `service.beta.kubernetes.io/alicloud-loadbalancer-address-type` to `intranet` for security.
```

File: en/deploy-on-alibaba-cloud.md [hunk 8]
```diff
@@ -163,29 +217,63 @@ The initial login user account and password:
...
+
+1. Refer to [Destroy a TiDB cluster](destroy-a-tidb-cluster.md) to delete the cluster.
+
+2. Destroy the ACK cluster by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    terraform destroy
+    ```	```
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
    ```
```

File: en/deploy-on-alibaba-cloud.md [hunk 9]
```diff
@@ -163,29 +217,63 @@ The initial login user account and password:
...
+
+If the Kubernetes cluster is not successfully created, the `destroy` operation might return an error and cannot be executed. In such cases, remove the Kubernetes resources from the local state manually:
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
If the Kubernetes cluster is not successfully created, the `destroy` operation might return an error and fail. In such cases, manually remove the Kubernetes resources from the local state:
```

File: en/deploy-on-alibaba-cloud.md [hunk 10]
```diff
@@ -163,29 +217,63 @@ The initial login user account and password:
...
+> **Note:**
+>
+> You have to manually delete the cloud disk used by the components in the Alibaba Cloud console.
 
 ## Configure
 
 ### Configure TiDB Operator
 
-You can adjust the `variables.tf` settings to configure TiDB Operator. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example,
+You can set variables in `terraform.tfvars` to configure TiDB Operator. Most configuration items can be modified after understanding the semantics based on the comments of the `variable`. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example:
```
Comment (yikeke) [inline en/deploy-on-alibaba-cloud.md]:
```suggestion
You can set the variables in `terraform.tfvars` to configure TiDB Operator. Most configuration items can be modified after you understand the semantics based on the comments of the `variable`. Note that the `operator_helm_values` configuration item can provide a customized `values.yaml` configuration file for TiDB Operator. For example:
```

**Comment by ran-huang** [comment]
@yikeke @DanielZhangQD PTAL

**Pull request merged**